clr.coreclr.props

```diff
@@ -83,6 +83,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetsUnix)' != 'true'">
+    <FeatureArrayStubAsIL Condition="('$(TargetArch)' == 'arm') or ('$(TargetArch)' == 'amd64') or ('$(TargetArch)' == 'arm64')">true</FeatureArrayStubAsIL>
```

Comment:
Nit: I would be nice to move `FeatureStubsAsIL` from the common place to here as well.